App/screens/group/group.tsx

```diff
@@ -137,7 +138,24 @@ class Group extends Component<Props, State> {
     )
   }
 
-  renderRow = ({ item }: ListRenderItemInfo<Item>) => {
+  sameUserAgain = (user: pb.IUser, previous: Item): boolean => {
+    if (!previous || !previous.type) {
+      return false
+    }
+    switch (previous.type) {
+      case 'message': {
+        if (user.name.length <= 0 || previous.data.user.name.length <= 0) {
+          return false
+        }
+        return user.name === previous.data.user.name
```

Comment:
Don't think we want to use `name` since it's not unique to a user. `address` would be it.

Reply:
ah yeah, good call